### DISCUSSION TITLE
literature notes抓取标题失败，读取tags无法判断英文词组

### DISCUSSION THREAD

**Discussion by fantasea2023** [aidenlx/obsidian-zotero#247] [Help]
<img width="707" alt="image" src="https://github.com/PKM-er/obsidian-zotlit/assets/149090816/2b341d1c-573b-4b01-8101-8c30bfc568ed">
<img width="1170" alt="image" src="https://github.com/PKM-er/obsidian-zotlit/assets/149090816/9aa85bb3-b24c-428b-898e-8441504fdea4">
<img width="341" alt="image" src="https://github.com/PKM-er/obsidian-zotlit/assets/149090816/f57386d9-2ecc-41c9-9e16-550c6c6bdd24">

如图，该篇文献标题应为“Work values across generations in China”，但是create literature note之后，插件没有抓取到正确的标题，模板的代码我有改过，但是与此应该无关。

另外，抓取tags时，插件无法判断英文词组，比如将“International management”识别为两个tags——“International”和“management”。

此外，希望能够在模板中区分“摘录区”和“笔记区”，摘录区的内容从pdf获取，会随着pdf注释的更新而更新，笔记区的内容则是自己手写的，不会随着update literature notes而更新。

非常感谢！

**Comment by InvisCat** [comment]
关于无法正确识别标题这个问题，我也遇到了。似乎是最近版本更新（包括better bibtex的更新）后新出的问题

**Comment by aidenlx** [comment]
tag可以参考[这段](https://zotlit.aidenlx.top/zh-CN/getting-started/basic-usage/template-config#note-properties-template:~:text=tags%3A%20%3C%25%3D%20JSON.stringify(it.tags)%20%25%3E)
```
tags: <%= JSON.stringify(it.tags) %>
```

**Comment by aidenlx** [comment]
> 如图，该篇文献标题应为“Work values across generations in China”，但是create literature note之后，插件没有抓取到正确的标题，模板的代码我有改过，但是与此应该无关。

你把模板发上来，你这个标题看起来是文件名的默认输出，不是正文的标题

**Comment by aidenlx** [comment]
这几个问题在 v2 里都不存在了：

- **tags**：`zt.tags` 是结构化的标签对象数组，每个有 `name`，不会再按空格把 `International management` 拆成两个标签。
- **标题**：笔记文件名由 `filename` 模板决定，正文标题直接用 `{{ zt.title }}`，两者互不影响。
- **摘录区 / 笔记区**：这正是 v2 的托管区域（managed region）—— `%%zt-managed%%` 标记内的内容每次更新都从 Zotero 重新渲染，标记之外是你自己写的，更新时不会被动到。

托管区域说明：https://zotlit.aidenlx.site/docs/concepts/literature-notes-and-the-managed-region

v2 beta 现已发布：https://zotlit.aidenlx.site/docs/install-zotlit
迁移指南：https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1

**Reply by InvisCat** [reply]
好的谢谢
